.github/workflows/wpt.yml

```
@@ -261,9 +261,9 @@ jobs:
           base: ${{ github.head_ref }}
           committer: Browser Automation Bot <browser-automation-bot@google.com>
           author: Browser Automation Bot <browser-automation-bot@google.com>
-          commit-message: 'test: update the expectations for PR'
-          title: 'test: update the expectations for PR'
-          body: 'Automatically generated by https://github.com/GoogleChromeLabs/chromium-bidi/blob/main/.github/workflows/wpt.yml'
+          commit-message: 'test: update the expectations for PR ${{ github.event.pull_request.number }}'
+          title: 'test: update the expectations for PR ${{ github.event.pull_request.number }}'
+          body: 'Update the expectations for PR ${{ github.event.pull_request.number }}, ${{ github.event.pull_request.title }}. \nAutomatically generated by https://github.com/GoogleChromeLabs/chromium-bidi/blob/main/.github/workflows/wpt.yml'
```

```suggestion
          body: 'Update the expectations for PR #${{ github.event.pull_request.number }} . \nAutomatically generated by https://github.com/GoogleChromeLabs/chromium-bidi/blob/main/.github/workflows/wpt.yml'
```

This should add as a link from the repo that way it will 
A) be clickable 
B) link it as a line in the original PR ("This PR was mentioned in ....");